mtaos_sim/Dockerfile

```diff
@@ -14,27 +14,9 @@ WORKDIR ${LSST_USER_HOME}
 
 # Download the required repos and checkout correct versions
 RUN cd ${AOS_REPOS} \
-    && cd ts_xml/ \
-    && git checkout 74ded62 sal_interfaces/MTM1M3/MTM1M3_Commands.xml \
-    && git checkout f57e9d1 sal_interfaces/MTM1M3/MTM1M3_Events.xml \
-    && cd .. \
-    && git clone --branch develop https://github.com/lsst-ts/ts_config_mttcs \
-    && cd ts_config_mttcs/ \
-    && git checkout 2924779 \
-    && cd .. \
-    && git clone --branch develop https://github.com/lsst-ts/ts_MTAOS.git \
+    && git clone --branch master https://github.com/lsst-ts/ts_MTAOS.git \
```

Comment:
you could also make the branch here a build argument with default value `master`

```
ARG mtaos_branch=master

RUN cd ${AOS_REPOS} \
        && git clone --branch ${mtaos_branch} https://github.com/lsst-ts/ts_MTAOS.git \
```

Reply:
This is intentional to expose the repo version (branch + commit) to be consistent with the README.md file for the maintenance.